#include "controller_scene_server_player_replied_handler.h"
#include "src/game_logic/thread_local/thread_local_storage.h"
#include "src/network/message_system.h"
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
void ControllerScenePlayerServiceRepliedHandler::EnterScene(entt::entity player,
	const ::ControllerEnterSceneRequest* request,
	::google::protobuf::Empty* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
}

void ControllerScenePlayerServiceRepliedHandler::LeaveScene(entt::entity player,
	const ::ControllerLeaveSceneRequest* request,
	::google::protobuf::Empty* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
}

void ControllerScenePlayerServiceRepliedHandler::LeaveSceneAsyncSavePlayerComplete(entt::entity player,
	const ::ControllerLeaveSceneAsyncSavePlayerCompleteRequest* request,
	::google::protobuf::Empty* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
}

